### DISCUSSION TITLE
Obtaining the IDs of detected sources

### DISCUSSION THREAD

**Discussion by cosmic-nori** [sep-developers/sep#107] [Q&A]
I want to label each sources detected by `sep.extract` in order to analyze time-series data on the same field. 
In the original SExtractor software, if we set `CHECKIMAGE_TYPE`  to `SEGMENTATION` (see https://sextractor.readthedocs.io/en/latest/Output.html), it can create segmentation map (FITS file), in which detected sources have their unique IDs. 
Can we also obtain such IDs of detected sources on memory with SEP? Thanks.

**Comment by kbarbary** [comment]
Sounds like you're looking for the `segmentation_map=True` argument in `sep.extract`?

**Reply by cosmic-nori** [reply]
Thank you for your reply. Yes, that is what I wanted to do. I missed the argument in your document until now.
I could obtain the segmentation map ID for each detected objects as a numpy array. Thank you again.
